uint16 pan_position
uint16 pan_speed
bool   pan_torque
uint16 tilt_position
uint16 tilt_speed
bool   tilt_torque